build-system/common/esbuild-babel.js

```diff
@@ -18,6 +27,56 @@ let CacheMessageDef;
  */
 let transformCache;
 
+/**
+ * @param {string} filename
+ * @param {string} contents
+ * @param {Object} babelOptions
+ * @return {!Promise<!CacheMessageDef>}
+ */
+async function transformInternal(filename, contents, babelOptions) {
+  debug('pre-babel', filename, contents);
+  const p = babel.transformAsync(contents, babelOptions);
+  const {code, map} = /** @type {!babel.BabelFileResult} */ (await p);
```

Comment:
Does trimming to one line work?
```suggestion
  const {code,map} = await babel.transformAsync(contents, babelOptions);
```